storage/badger/operation/version_beacon_test.go

```diff
@@ -0,0 +1,82 @@
+package operation
+
+import (
+	"testing"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+func TestResults_IndexByServiceEvents(t *testing.T) {
+	unittest.RunWithBadgerDB(t, func(db *badger.DB) {
+
+		vb1 := unittest.VersionBeaconFixture(2)
+		vb2 := unittest.VersionBeaconFixture(2)
+		vb3 := unittest.VersionBeaconFixture(2)
```

Comment:
Let's make the three version beacon to be different, otherwise checking equal to one of them would always return true.
```suggestion
		vb1 := unittest.VersionBeaconFixture(1)
		vb2 := unittest.VersionBeaconFixture(2)
		vb3 := unittest.VersionBeaconFixture(3)
```

Reply:
We use random versions for actual requirement table, this argument is just a size of the table

Reply:
I see. The randomness is quite deep in the `VersionControlRequirementFixture`. 

I think it's still good to use `1,2,3`, as the real scenario would most likely have the increasing number of VersionControlRequirement.

Reply:
It certainly won't hurt - so changed as requested